be/src/formats/parquet/stored_column_reader.cpp

```diff
@@ -149,7 +148,8 @@ class RequiredStoredColumnReader : public StoredColumnReader {
     }
 
 private:
-    Status _next_page();
+    Status _next_selected_page(size_t records_to_read, size_t* records_to_skip);
```

Comment:
What does this function mean?

Reply:
> What does this function mean?

It aims to find a next valid page which satisfy the filter condition (not all rows are skipped).